### ISSUE TITLE
Compilation error in 0.7.0/master with Crystal 0.24.1

### ISSUE THREAD

**Issue by develop7** [ysbaddaden/prax.cr#72]
Repro: run `make`

Expected: compilation succeeds
Instead: compilation fails on

https://github.com/ysbaddaden/prax.cr/blob/42bd85c446a26bb860ced7956f3b72597d719a06/src/prax.cr#L18

with following error:

```
develop7@t-rex ~/s/prax.cr (master)>  make
mkdir -p bin
/home/develop7/.linuxbrew/bin/crystal build /home/develop7/src/prax.cr/src/prax.cr -o bin/prax-binary
Syntax error in src/prax.cr:18: unexpected token: self (parentheses are mandatory for macro arguments)

  macro def self.version_string : String
            ^

make: *** [Makefile:25: all] Error 1
```

**Comment by ysbaddaden** [comment]
Oh, fixed in 8770275f78263c0618f52b4b5147ac2b11406a4b !

**Comment by develop7** [comment]
That fixes it, thanks!
